npm_and_yarn/spec/fixtures/projects/yarn/yarnrc_global_registry_with_space/package.json

```diff
@@ -0,0 +1,25 @@
+{
+    "name": "{{ name }}",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no\\ test\ specified\" && exit 1",
```

Comment:
Same with the missing double slash. I double checked and that makes the script unrunnable, you get an error like this:

```
$ yarn test
error An unexpected error occurred: "/path/to/package.json: Bad escaped character in JSON at position 121".
```

No big deal but best to use sane fixtures in our specs!

Reply:
Yup addressed them. Thanks for review @deivid-rodriguez